src/pages/post-test-summary/post-test-summary.ts

```diff
@@ -125,7 +125,7 @@ export class PostTestSummaryPage {
   private backToJournal() {
     this.faultStore.reset();
     this.summaryMetadata.reset();
-    this.navCtrl.popToRoot();
+    this.navCtrl.popTo(this.navCtrl.getByIndex(1));
```

Comment:
any specific reason for changing this?

Reply:
In order to show the back button on the Journal page, I had to remove some code which pruned the Welcome page out of the navigation stack. This meant that `popToRoot` was sending us to the welcome page after test completion.